### ISSUE TITLE
 schema report preview error

### ISSUE THREAD

**Issue by AlenKelemen** [jumpinjackie/mapguide-rest#178]
GET /library/{resourcePath}.FeatureSource/preview returns:

`PHP Warning:  DOMDocument::load(): I/O warning : failed to load external entity &quot;C:\Program Files\OSGeo\MapGuide\Web\www\schemareport\displayschema.xsl&quot; in C:\Program Files\OSGeo\MapGuide\Web\www\schemareport\displayschema.php on line 110
PHP Warning:  XSLTProcessor::importStylesheet(): compilation error in C:\Program Files\OSGeo\MapGuide\Web\www\schemareport\displayschema.php on line 114
PHP Warning:  XSLTProcessor::importStylesheet(): xsltParseStylesheetProcess : empty stylesheet in C:\Program Files\OSGeo\MapGuide\Web\www\schemareport\displayschema.php on line 114
PHP Warning:  XSLTProcessor::transformToXml(): No stylesheet associated to this object in C:\Program Files\OSGeo\MapGuide\Web\www\schemareport\displayschema.php on line 137`

I have Mapguide 3.0.0.8701. It seems that something is missing in those files. Sometimes it works OK so I think it is depending on the feature source content.

**Comment by jumpinjackie** [comment]
The preview is just a redirect to the existing schema report that comes with the MapGuide install (as evidenced by the paths in your provided stack trace). So the problem isn't with mapguide-rest.

Does this consistently fail on some feature sources? If so, can it be isolated to a particular FDO provider, or certain properties in its class definition?

**Comment by AlenKelemen** [comment]
Yes, the same schema report problem also occurs in Maestro.
I tested  schema report with different feature sources, different MG installs and different computers.
I did not found this kind of problem on other machines (but found some others :-) ) and MG installations.
On my machine error occurs randomly with no correlation to feature source or FDO.
I will close this issue as now I believe that its cause must be somewhere else.
